kbfsgit/runner.go

```diff
@@ -253,6 +253,25 @@ func (r *runner) printDoneOrErr(
 	}
 }
 
+func (r *runner) isManagedByApp() bool {
+	switch r.h.Type() {
+	case tlf.Public:
+		// Public TLFs are never managed by the app.
+		return false
+	case tlf.SingleTeam:
+		// Single-team TLFs are always managed by the app.
+		return true
+	case tlf.Private:
+		// Only single-user private TLFs are managed by the app.  So
```

Comment:
What about single-user private TLFs that haven't yet been transitioned to implicit teams?

Reply:
What's the concern?  This function will correctly return `true` in that case as well, right?

Reply:
Yes, single-user private repos are managed by the app/CLI (i.e., the "personal" section of the GUI), along with team repos.

Reply:
Ah, I see now what is meant by "managed". Thanks!